### DISCUSSION TITLE
hi~ Somebody help me~

### DISCUSSION THREAD

**Discussion by smartkuk** [ExistentialAudio/BlackHole#601] [Q&A]
Hello~
I am a newbie using black holes.
Recently, I discovered a strange phenomenon, that when deleting a file in the iterm terminal, when a prompt asking yes or no occurs, recording through blackhole is forcibly stopped.
I was wondering if there is a way to solve this similar problem
Note that this question has been directed to the translator.

**Comment by devinroth** [comment]
Please give me steps on how to replicate this issue.

Devin

**Comment by arangari** [comment]
Hello,

I am another newbie....

I am user of Mac + AirPods Pro . when I installed blackhole 2 - the connection to AirPods is not stable. I am not sure if it is an issue in configuration or anything else.

Additionally, I am a user of Microsoft OneNote on Mac.  I was not able to define "audio-in" for the application.
Need help.
